src/audio/up_down_mixer/up_down_mixer.c

```diff
@@ -38,6 +40,37 @@ DECLARE_SOF_RT_UUID("up_down_mixer", up_down_mixer_comp_uuid, 0x42f8060c, 0x832f
 DECLARE_TR_CTX(up_down_mixer_comp_tr, SOF_UUID(up_down_mixer_comp_uuid),
 	       LOG_LEVEL_INFO);
 
+#define ADSP_BUILD_INFO_FORMAT 0
+
+struct sof_module_api_build_info udm_build_info __attribute__((section(".buildinfo"))) = {
+	ADSP_BUILD_INFO_FORMAT,
+	{
+		((0x3FF & 5)  << 20) |
+		((0x3FF & 0) << 10) |
+		((0x3FF & 0)  << 0)
+	}
+};
+
+extern struct module_interface up_down_mixer_interface;
```

Comment:
this forward really needed? Maybe move declaration of sof_module_api_build_info to the end of the file?